ImperatorToCK3/Data_Files/blankMod/output/localization/ir_names_l_english.yml

```diff
@@ -0,0 +1,1940 @@
+ Psherenptah:0 "Psherenptah"
+ Psammuthes:0 "Psammuthes"
+ Theocritus:0 "Theocritus"
+ Therapon:0 "Therapon"
+ Xanthos:0 "Xanthos"
+ Xenon:0 "Xenon"
+ Zeteres:0 "Zeteres"
+ Ahmes:0 "Ahmes"
+ Ahmose:0 "Ahmose"
+ Alarus:0 "Alarus"
+ Amasus:0 "Amasus"
+ Ameny:0 "Ameny"
+ Bakenrenef:0 "Bakenranef"
+ Cleomenes:0 "Cleomenes"
+ Euclid:0 "Euclid"
+ Harsiotef:0 "Harsiotef"
+ Henut:0 "Henut"
+ Ibi:0 "Ibi"
+ Khabash:0 "Khabash"
+ Lagus:0 "Lagus"
+ Nectanebo:0 "Nectanebo"
+ Kheperkare:0 "Kheperkhare"
+ Neferkare:0 "Neferkare"
+ Nehi:0 "Nehi"
+ Osorkon:0 "Osorkon"
+ Pami:0 "Pami"
+ Petubastis:0 "Petubastis"
+ Teos:0 "Teos"
+ # Egyptian Female
+ Agape:0 "Agape"
+ Agathoclea:0 "Agathokleia"
+ Amanishabheto:0 "Amanishakheto"
+ Antigone:0 "Antigone"
+ Apame:0 "Apame"
+ Aristomache:0 "Aristomache"
+ Artemisia:0 "Artemisia"
+ Aspacia:0 "Aspacia"
+ Athenais:0 "Athenais"
+ Charis:0 "Charis"
+ Cleopatra:0 "Cleopatra"
+ Cynna:0 "Cynna"
+ Demetria:0 "Demetria"
+ Elpis:0 "Elpis"
+ Eunice:0 "Eunice"
+ Eurydice:0 "Eurydice"
+ Euthalia:0 "Euthalia"
+ Nysa:0 "Nysa"
+ Oenanthe:0 "Oenanthe"
+ Olympia:0 "Olympias"
+ Phila:0 "Phila"
+ Phile:0 "Phile"
+ Sostrate:0 "Sostrate"
+ Stratonice:0 "Stratonice"
+ Thais:0 "Thaïs"
+ Theophila:0 "Theophila"
+ Thetima:0 "Thetima"
+ Thessaloniki:0 "Thessaloniki"
+ Thorakis:0 "Thorakis"
+ Timo:0 "Timo"
+ Arsinoe:0 "Arsinoe"
+ Lysandra:0 "Lysandra"
+ Kiya:0 "Kiya"
+ Qelhata:0 "Qelhata"
+ # Egyptian dynasties
+ dynn_Ahmid:0 "Ahmid"
+ dynn_Galestid:0 "Galestid"
+ dynn_Kenamid:0 "Kenamid"
+ dynn_Naravid:0 "Naravid"
+ dynn_Penamid:0 "Penamid"
+ dynn_Senuid:0 "Senuid"
+ dynn_Setnid:0 "Setnid"
+ dynn_Arganid:0 "Arganid"
+ dynn_Psammid:0 "Psammid"
+ dynn_Philothid:0 "Philothid"
+ dynn_Timolid:0 "Timolid"
+ dynn_Zosid:0 "Zosid"
+ dynn_Phileid:0 "Phileid"
+ dynn_Nicodemid:0 "Nicodemid"
+ dynn_Nefarid:0 "Nefarid"
+ dynn_Bastid:0 "Bastid"
+ dynn_Xeneid:0 "Xeneid"
+ dynn_Ezanid:0 "Ezanid"
+ dynn_Alarid:0 "Alarid"
+ dynn_Harsiotid:0 "Harsiotid"
+ dynn_Henid:0 "Henid"
+ dynn_Ibid:0 "Ibid"
+ dynn_Khabbashid:0 "Khabbashid"
+ dynn_Amasid:0 "Amasid"
+ dynn_Nectanebid:0 "Nectanebid"
+ dynn_Petubasid:0 "Petubasid"
+ dynn_Djedhid:0 "Djedhid"
+ dynn_Tachyid:0 "Tachyid"
+ dynn_Magid:0 "Magid"
+ # Numidian Male
+ Bomilcar:0 "Bomilcar"
+ Gulussa:0 "Gulussa"
+ Iampsas:0 "Iampsas"
+ Jugurtha:0 "Jugurtha"
+ Nabdalsa:0 "Nabdalsa"
+ Sypax:0 "Sypax"
+ Syphax:0 "Syphax"
+ Tacfarinas:0 "Tacfarinas"
+ Tychaeus:0 "Tychaeus"
+ Vermina:0 "Vermina"
+ Gala:0 "Gala"
+ Amastan:0 "Amastan"
+ # Numidian Female
+ Salammbo:0 "Salammbo"
+ Sophonisba:0 "Sophonisba"
+ Panisba:0 "Panisba"
+ Ena:0 "Ena"
+ Palema:0 "Palema"
+ Synno:0 "Synno"
+ Narva:0 "Narva"
+ Adhierno:0 "Adhierno"
+ Ballanba:0 "Ballanba"
+ # Numidian Dynasties
+ dynn_Adherbid:0 "Adherbid"
+ dynn_Massinid:0 "Massinid"
+ dynn_Tacfarinid:0 "Tacfarinid"
+ dynn_Syphid:0 "Syphid"
+ dynn_Verminid:0 "Verminid"
+ dynn_Galid:0 "Galid"
+ dynn_Jugurthid:0 "Jugurthid"
+ 
+ mercenary_company_legion:0 "Roman Legion"
```

Comment:
This should probably be in a separate file.

Reply:
Agreed, I'll solve it in the next pull where I localize also the names of Byzantine converter cultures.